e2e/jest.test-setup.ts

```diff
@@ -42,10 +42,10 @@ beforeAll(async () => {
     // to improve page load performance, block network requests unrelated to application.
     try {
       if (host === 'www.google-analytics.com'
-         || host === 'accounts.youtube.com'
-         || host === 'static.zdassets.com'
-         || host === 'play.google.com'
-         || request.url().endsWith('content-security-index-report')) {
+           || host === 'accounts.youtube.com'
```

Comment:
can we revert this to original indent

Reply:
yes. it was lint that formatted this way.